packages/webview_flutter/tizen/src/buffer_pool.h

```diff
@@ -0,0 +1,49 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_PLUGIN_WEBVIEW_FLUTTER_TIZEN_WEBVIEW_BUFFER_POOL_H_
+#define FLUTTER_PLUGIN_WEBVIEW_FLUTTER_TIZEN_WEBVIEW_BUFFER_POOL_H_
+
+#include <tbm_surface.h>
+
+#include <memory>
+#include <mutex>
+#include <string>
+#include <vector>
+
+class BufferUnit {
+ public:
+  explicit BufferUnit(int index, int width, int height);
+  ~BufferUnit();
+  void Reset(int width, int height);
+  bool Allocate();
+  void Release();
```

Comment:
These methods don't allocate or deallocate memory. I suggest you consider another names